lib/parallel_cucumber/event_broadcaster.js

```diff
@@ -0,0 +1,103 @@
+var fireBeforeFeatures = true,
```

Comment:
@ViacheslauBushyla can we move the two Cucumber files into their own directory.  E.g. `lib/cucumber/runtime`.